src/utils/lightning/index.ts

```diff
@@ -1109,8 +1110,9 @@ export const addPeers = async ({
 		// No need to add Blocktank peer if geo-blocked.
 		if (!geoBlocked) {
 			// Set Blocktank node uri array if able.
-			blocktankNodeUris =
-				getBlocktankStore()?.info?.nodes[0]?.connectionStrings ?? [];
+			for (const node of getBlocktankStore()?.info?.nodes ?? []) {
+				blocktankNodeUris.push(...(node?.connectionStrings ?? []));
+			}
```

Comment:
I find this a bit hard to read. The types say none of this can be undefined so this can be:
```suggestion
			const nodes = getBlocktankStore().info.nodes;
			for (const node of nodes) {
				blocktankNodeUris.push(...node.connectionStrings);
			}
```

Reply:
I thought maybe those precautions are here for a reason?
But I agree, I will remove them